l10n_it_fatturapa_export_zip/wizard/export_fatturapa.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+import base64
+import io
+import zipfile
+from datetime import datetime
+from odoo import models, api, fields, _
+from odoo.exceptions import UserError
+
+
+class WizardAccountInvoiceExport(models.TransientModel):
+    _name = "wizard.fatturapa.export"
+
+    @api.model
+    def _default_name(self):
+        return "%s_%s" % (
+            _("E-invoice-export"), datetime.now().strftime('%Y%m%d%H%M'))
+
+    data = fields.Binary("File", readonly=True)
+    name = fields.Char('Filename', default=_default_name)
+
+    @api.multi
+    def export_zip(self):
+        attachments = []
```

Comment:
Why do you need a list? You can assign here the result of `browse` directly.
Moreover:
```suggestion
        self.ensure_one()
        attachments = []
```

Reply:
> Why do you need a list?

@SimoRubi beacause `active_model` can be both `fatturapa.attachment.in` and `fatturapa.attachment.out`

Reply:
But you can write 
```
attachments = self.env[self._context['active_model']].browse(self._context['active_ids'])
```
There would be a problem if the model of its elements was checked somewhere, but using a list wouldn't solve it anyway

Reply:
Right, done